xml/geo_concept_i.xml

```diff
@@ -228,15 +228,19 @@
    <term>Ticket Failover</term>
    <listitem>
     <para>
-     If the ticket gets lost, which means other booth instances do not hear
-     from the ticket owner in a sufficiently long time, one of the remaining
-     sites will acquire the ticket. This is what is called ticket failover. If
-     the remaining members cannot form a majority, then the ticket cannot fail
-     over (if managed by booth).
+     After you have initially granted a normal ticket to a site, the booth
```

Comment:
"an automatic" instead of "normal". Also, perhaps "Booth" instead of "the booth mechanism"